generate_variant_annotation.py

```diff
@@ -247,56 +122,72 @@ def main():
     )
 
     # Sort columns
-    col_order = ['locus_GRCh38', 'chrom_b37', 'pos_b37', 'chrom_b38', 'pos_b38',
-                 'ref', 'alt', 'allele_type', 'vep', 'rsid', 'af', 'cadd']
-    ht = ht.select(*col_order)
-
-    # Persist as writing twice would cause re-computation
-    ht = ht.persist()
+    col_order = [
+        'locus_GRCh38', 'chrom_b38', 'pos_b38',
+        'chrom_b37', 'pos_b37',
+        'ref', 'alt', 'allele_type', 'vep', 'rsid', 'af', 'cadd'
+    ]
 
     # Repartition and write parquet file
     (
-        ht.to_spark(flatten=False)
-          .repartition(out_partitions)
-          .write.parquet(out_parquet)
+        ht
+        .select(*col_order)
+        .to_spark(flatten=False)
+        # .repartition(OUT_PARTITIONS)
+        .coalesce(OUT_PARTITIONS)
+        .write.mode('overwrite').parquet(out_parquet)
+        # .write.format('json').mode('overwrite').option('compression', 'gzip').save(out_parquet.replace('.parquet', '.json.gz'))
```

Comment:
Do we need the commented out lines (`repartition...` and `.write.format('json')...`)?